lib/ansible/modules/network/ovs/openvswitch_db.py

```diff
@@ -125,8 +144,15 @@ def map_config_to_obj(module):
     match = re.search(r'^' + module.params['col'] + r'(\s+):(\s+)(.*)$', out, re.M)
 
     col_value = match.group(3)
+
+    # Map types require key argument
+    has_key = module.params['key'] is not None
+    is_map = col_value[0] == '{'
```

Comment:
is_map should be done using global re 'MAP_RE' so we have one way to identify map in a column value.

Reply:
Done